frontend/src/modules/userManagement/templates/addUser.hbs

```diff
@@ -3,15 +3,25 @@
     {{t 'app.adduserformtitle'}}
   </div>
   <form class="addUser forms" action="/api/user" method="post" autocomplete="off">
+      <div class="form-group">
+        <label for="firstName">{{t 'app.firstname'}} <span class="req">*</span></label>
```

Comment:
label's for attribute must match an input element id. Since we don't have any id's I would suggest to remove the for attribute at all. The label may wrap the input element so that it sets focus if a user clicks on the label text.